Got an error on mobile when I edited a page.

Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        |Ubuntu|
|GROWI     |5.0.0|
|node.js   |v14.19.1|
|npm       |6.14.16|
|Using Docker|no|
|Using [growi-docker-compose][growi-docker-compose]|no|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose


### Client

| item     | version |
| ---      | --- |
|OS        |Manjaro and Android|
|browser   |Chrome|



How to reproduce? (再現手順)
---------------------------

1. Edit a page on mobile phone
1. Got an error message following below that:

![Screenshot_20220403_224303](https://user-images.githubusercontent.com/8959761/161430993-a4f18804-a2b1-4b20-b90b-f92dd94d18ea.png)


```
react_devtools_backend.js:3973 Error: The target 'grw-page-editor-mode-manager-hackmd-button' could not be identified in the dom, tip: check spelling
    at T (vendors.c18fdd0d2de2eb7a13fb.chunk.js:2:5873511)
    at j (vendors.c18fdd0d2de2eb7a13fb.chunk.js:2:5873719)
    at t.n.updateTarget (vendors.c18fdd0d2de2eb7a13fb.chunk.js:2:5860885)
    at t.n.componentDidMount (vendors.c18fdd0d2de2eb7a13fb.chunk.js:2:5857190)
    at la (vendors.c18fdd0d2de2eb7a13fb.chunk.js:2:5516090)
    at vl (vendors.c18fdd0d2de2eb7a13fb.chunk.js:2:5533976)
    at t.unstable_runWithPriority (vendors.c18fdd0d2de2eb7a13fb.chunk.js:2:5970600)
    at $i (vendors.c18fdd0d2de2eb7a13fb.chunk.js:2:5477750)
    at yl (vendors.c18fdd0d2de2eb7a13fb.chunk.js:2:5530448)
    at il (vendors.c18fdd0d2de2eb7a13fb.chunk.js:2:5526602)
```



What is the expected result? (期待される動作)
-------------------------------------------

- The error will not be displaed

--- a/packages/app/src/components/Navbar/PageEditorModeManager.jsx
+++ b/packages/app/src/components/Navbar/PageEditorModeManager.jsx
@@ -87,25 +87,27 @@ function PageEditorModeManager(props) {
           />
         )}
         {(!isDeviceSmallerThanMd || editorMode === EditorMode.View) && showHackmdBtn && (
-          <PageEditorModeButtonWrapper
-            editorMode={editorMode}
-            isBtnDisabled={isBtnDisabled || !isHackmdEnabled}
-            onClick={pageEditorModeButtonClickedHandler}
-            targetMode={EditorMode.HackMD}
-            icon={<i className="fa fa-file-text-o" />}
-            label={t('hackmd.hack_md')}
-            id="grw-page-editor-mode-manager-hackmd-button"
-          />
+          <>
+            <PageEditorModeButtonWrapper
+              editorMode={editorMode}
+              isBtnDisabled={isBtnDisabled || !isHackmdEnabled}
+              onClick={pageEditorModeButtonClickedHandler}
+              targetMode={EditorMode.HackMD}
+              icon={<i className="fa fa-file-text-o" />}
+              label={t('hackmd.hack_md')}
+              id="grw-page-editor-mode-manager-hackmd-button"
+            />
+            { !isHackmdEnabled && (
+              <UncontrolledTooltip placement="top" target="grw-page-editor-mode-manager-hackmd-button" fade={false}>
+                {t('hackmd.not_set_up')}
+              </UncontrolledTooltip>
+            )}
+          </>
         )}
       </div>
       {isBtnDisabled && (
         <UncontrolledTooltip placement="top" target="grw-page-editor-mode-manager" fade={false}>
           {t('Not available for guest')}
-        </UncontrolledTooltip>
-      )}
-      {(!isDeviceSmallerThanMd || editorMode === EditorMode.View) && showHackmdBtn && !isHackmdEnabled && (
-        <UncontrolledTooltip placement="top" target="grw-page-editor-mode-manager-hackmd-button" fade={false}>
-          {t('hackmd.not_set_up')}
         </UncontrolledTooltip>
       )}
     </>

